src/Resources/config/services.yml

```diff
@@ -1,4 +1,10 @@
 services:
-#    salesforce.example:
-#        class: Oowlish\Example
-#        arguments: ["@service_id", "plain_value", %parameter%]
+    GuzzleHttp\Client:
```

Comment:
I'm not sure about this indentation. It seems weird to me.

Reply:
Yes, for sure. I think that I'm wrong. Is it possible to put that validation on StyleCI to yaml files?

Reply:
I don't think so.

Reply:
I already put this settings on the `.editorconfig` file. Thx!